### ISSUE TITLE
PVR causes CEC reset loop

### ISSUE THREAD

**Issue by c00ntact** [OpenELEC/OpenELEC.tv#3790]
Hardware: IMX6/CuboxTv with Samsung TV. Separate Mythtv backend running 0.27

After upgrading to 5.0.0 I am encountering the following:

Once I enable live TV (mythtv PVR add-on) the CEC status seems to reset and cause the XBMC to restart. I have tried disabling CEC on all devices and still run into the same problem. Essentially I am unable to enable liveTV without XBMC restarting after about 10 seconds. This behavior is independent of the connected hardware. Any help would be greatly appreciated

**Comment by c00ntact** [comment]
please let me know if you need more details such as the logs etc. in order to provide comments/help

**Comment by stefansaraev** [comment]
sure, we need more details. [see](https://github.com/OpenELEC/OpenELEC.tv/blob/master/CONTRIBUTING.md)
thanks.

**Comment by BigYesh** [comment]
I have experienced what appears to be the same problem. I have a fresh install of OpenELEC 5.0.1 on a Raspberry Pi2. After installation I have done the following:

1) Enable SSH
2) Turn on debug logging and detailed component logging and selected all options
3) Enabled MythTV PVR addon

Kodi crashes then while loading channels (log http://sprunge.us/VMbM). It restarts and then freezes on the main screen (log http://sprunge.us/XQai)

Let me know if there is anything else you need.

**Comment by BigYesh** [comment]
I have also tried disabling CEC. In that case it crashes while loading channels (http://sprunge.us/KRUV), starts again and crashes slightly later (http://sprunge.us/GOPS) and then restarts again and crashes on the main screen (http://sprunge.us/iWRW).

**Comment by c00ntact** [comment]
Upgrading to 5.0.1 has actually solved my issue. Thank you

**Comment by shaun2029** [comment]
I have OpenELEC 5.0.2 on a RaspberryPi 2B. 

When running the MythTV PVR addon I get crashing and instability. To fix this issue I have patched the kodi-pvr-addon with the cppmyth 25c1170. This has fixed the crashing.

**Comment by c00ntact** [comment]
HI Shaun,

I assume that you had to build the cppmyth addon. Would you mind helping me through the steps on Openelec. Which build libraries do I need to downlaod to compile from source and successfully run the cmake command outlined in the (cppmyth)  README file.

While my issue is resolved the current mytht-pvr addon seems not as smooth as on my other platforms.

Thank you

**Comment by shaun2029** [comment]
I will upload a build for imx6 if you are interested. 

The kodi-pvr-addons are being updated and the latest official/test builds may have improved MythTV support.

Building:
I compiled the kodi-pvr-addon that builds the MythTV and cppmyth. To build the kodi-pvr-addon I build OpenELEC.tv.

Building OpenELEC.tv is really easy if you are familiar with Linux. It downloads and installs and configures the packages you need. I am using Ubuntu 14.10 64 bit.

I have a RaspberryPI2 and followed the instructions here:
http://wiki.openelec.tv/index.php/Building_and_Installing_OpenELEC_for_Raspberry_Pi

The instructions are good and up to date. There will be more info on the forums.

Once you have a compile complete, takes 2.5 hours to build on my SSD system, you can patch the source code in the build folder or experiment with the files in the package folder called package.mk. 

Each package has a folder with the package name in the packages folder. The build instructions are in a package.mk file. The patches for the package are in the patches folder.

For kodi-pvr-addons:

packages/mediacenter/
├── kodi-pvr-addons
│   ├── package.mk
│   └── patches
│       ├── kodi-pvr-addons-buildfix.patch
│       └── kodi-pvr-addons-mythtv_cppmyth_update_25c1170.patch

You can use GitHub to for the OpenELEC.tv repository, use git to clone it and off you go.
You can also look for test builds for your device.

**Comment by camelreef** [comment]
Was this merged into 5.0.6?

I'm experiencing the same issue.

Nico

**Comment by c00ntact** [comment]
I am re-experiencing the same problem again after upgrading to 5.0.6!

**Comment by atarijeff** [comment]
Subscribed. I am having the exact same issue with 5.0.6.

**Comment by camelreef** [comment]
I have built using the 25c1170 patch, and it looks stable.

Also, based on @stefansaraev pull request, I have built this:

http://youplala.net/~nico/kodi/OpenELEC-RPi2.arm-devel-20150326210006-r20131-g9aa7b4c.tar

I will test once my SO is done watching TV!

**Comment by camelreef** [comment]
Testing positive

**Comment by ChrisHolcomb** [comment]
Same issues on RPi 2 and OpenElec 5.0.6 and MythTV PVR addin.
